### PR TITLE
Linking post favorites + fix

### PR DESCRIPTION
**Pull request by jasonvanmalder**

### CODE REVIEW

**Comment by jasonvanmalder** [comment]
[FE: Liaison des favoris](https://trello.com/c/U7YcajKX/171-fe-liaison-des-favoris)

File: src/components/common/post/Comment.js [hunk 1]
```diff
@@ -154,6 +154,10 @@ class Comment extends Component {
     this._isMounted = false
     clearInterval(this.interval)
   }
+  
+  componentDidUpdate = () => {
+    console.log('oui')
```
Comment (Sylv11) [inline src/components/common/post/Comment.js]: Don't you have to remove that `console.log('oui')` ?

Comment (Sylv11) [inline src/components/common/post/Comment.js]:
```suggestion
```

File: src/components/common/post/Comment.js [hunk 2]
```diff
@@ -154,6 +154,10 @@ class Comment extends Component {
...
+  }
```
Comment (Sylv11) [inline src/components/common/post/Comment.js]:
```suggestion
```

File: src/components/common/post/Comment.js [hunk 3]
```diff
@@ -154,6 +154,10 @@ class Comment extends Component {
     this._isMounted = false
     clearInterval(this.interval)
   }
+  
+  componentDidUpdate = () => {
```
Comment (Sylv11) [inline src/components/common/post/Comment.js]:
```suggestion
```

**Pull request merged**